Allow setting "pointabilities" for Lua raycasts (#14390)
minetest/minetest: doc/lua_api.md

    * Returns the position of the blocking node when `false`
    * `pos1`: First position
    * `pos2`: Second position
* `minetest.raycast(pos1, pos2, objects, liquids, pointabilities)`: returns `Raycast`
    * Creates a `Raycast` object.
    * `pos1`: start of the ray
    * `pos2`: end of the ray
    * `objects`: if false, only nodes will be returned. Default is `true`.
    * `liquids`: if false, liquid nodes (`liquidtype ~= "none"`) won't be
                 returned. Default is `false`.

>     * `pointabilities`: Contains lists to override the `pointable` property of
>       nodes and objects. Uses the same format as the `pointabilities` property
>       of item definitions. Default is `nil`.

Reviewer: clearer imo
```suggestion
    * `pointabilities`: Allows overriding the `pointable` property of
      nodes and objects. Uses the same format as the `pointabilities` property
      in the item definition. Default is `nil`.
```

Author: ```suggestion
    * `pointabilities`: Allows overriding the `pointable` property of
      nodes and objects. Uses the same format as the `pointabilities` property
      of item definitions. Default is `nil`.
```

"the item definition" seems to imply that there is one currently relevant item definition, which there isn't

    * `objects`: if false, only nodes will be returned. Default is `true`.
    * `liquids`: if false, liquid nodes (`liquidtype ~= "none"`) won't be
                 returned. Default is `false`.

>     * `pointabilities`: Contains lists to override the `pointable` property of

Reviewer 2: In the function description above, the `pointabilities` parameter is missing.
Should be
```md
 * `minetest.raycast(pos1, pos2, objects, liquids, pointabilities)`: returns `Raycast`
```

Author: Oops, thank you.
